pages/deployment/[uuid].tsx

```diff
@@ -187,30 +188,50 @@ export default function Deployment() {
   return (
     <>
       <Wrapper title={`Deployment: ${deployment.metadata.highLevelDetails.name}`} page='deployment'>
-        {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.Zip && (
-          <Box sx={{ textAlign: 'right', pb: 3 }}>
-            <Button variant='outlined' color='primary' startIcon={<Info />} onClick={handleClickOpen}>
-              Show download commands
-            </Button>
-          </Box>
-        )}
-        {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.ModelCard && (
-          <Box sx={{ pb: 2 }}>
-            <Alert
-              severity='info'
-              sx={{
-                width: 'fit-content',
-                m: 'auto',
-                backgroundColor: '#0288d1',
-                color: '#fff',
-                '& .MuiAlert-icon': {
-                  color: '#fff',
-                },
-              }}
+        {deployment && (
+          <Stack direction='row' alignItems='center' justifyContent='space-between'>
+            <Button
+              variant='text'
+              color='primary'
+              sx={{ pb: 3 }}
```

Comment:
Instead of doing this you should use the `spacing` prop on `Stack`. `spacing={3}` should do what you want, but the benefit is that you won't have to keep manually adding `sx={{ pb: 3 }}` to every button bar the last one.

Reply:
The pb is the padding below the row of buttons, not between the buttons